subsys/bluetooth/host/Kconfig

```diff
@@ -509,6 +509,14 @@ config BT_KEYS_SAVE_AGING_COUNTER_ON_PAIRING
 	  time a successful pairing occurs. This increases flash wear out but offers
 	  a more correct finding of the oldest unused pairing info.
 
+config BT_SMP_MIN_ENC_KEY_SIZE
+	int "Minimum encrypytion key size accepted in octets"
```

Comment:
```suggestion
	int "Minimum encryption key size accepted in octets"
```

Reply:
Nice catch :D Sounds like I was adding some kind of python encryption